index.d.ts

```diff
@@ -0,0 +1,122 @@
+/* =================== USAGE ===================
+
+    import express = require("express");
+    var app = express();
+
+ =============================================== */
+
+/// <reference types="./core" />
+/// <reference types="serve-static" />
```

Comment:
these lines are for old versions of TypeScript, presumably v1.x
I believe we can get rid of them, as well as the installed `@types/*` dev dependencies on the next step.

Reply:
I came to conclusion that we don't need tripple slash directives here.
I'm removing it together with the `@types/...` dependencies they refer to